app/code/Magento/Backend/i18n/en_US.csv

```diff
@@ -333,7 +333,7 @@ Debug,Debug
 "Add Block Names to Hints","Add Block Names to Hints"
 "Template Settings","Template Settings"
 "Allow Symlinks","Allow Symlinks"
-"Warning! Enabling this feature is not recommended on production environments because it represents a potential security risk.","Warning! Enabling this feature is not recommended on production environments because it represents a potential security risk."
+"<strong style="color:red">Warning!</strong> Enabling this feature is not recommended on production environments because it represents a potential security risk.","<strong style="color:red">Warning!</strong> Enabling this feature is not recommended on production environments because it represents a potential security risk."
```

Comment:
I believe you have to escape double quotes inside double quotes in translation CSV files. For example, you can check Catalog translation file.

Reply:
@kalpmehta : Thank you very much. I have fixed it. You can check it again. I tested it and it can be translated.

Reply:
@edenduong Can you please use double-quotes? Magento uses double-quotes in all other places, you can escape it by having two double-quotes like `<strong style=""color:red"">` I believe.

Reply:
Thank you for your suggestion. I have changed the source code. Please check it again. Thanks!